docs/logging-usage.rst

```diff
@@ -236,8 +236,8 @@ Create a BigQuery sink:
    >>> client = logging.Client()
    >>> sink = client.sink(
    ...     "robots-bq",
-   ...     filter='log:apache-access AND textPayload:robot')
-   >>> sink.bigquery_dataset = "projects/my-project/datasets/my-dataset"
+   ...     'log:apache-access AND textPayload:robot',
+   ...     "projects/my-project/datasets/my-dataset")
```

Comment:
Should be `bigquery.googleapis.com/projects/projects/my-project/datasets/my-dataset`.